addons/l10n_ie/data/template/account.account-ie.csv

```diff
@@ -1,104 +1,738 @@
+"l10n_ie_account_21224","21224","Trademarks and franchises","asset_fixed","False"
+"l10n_ie_account_212251","212251","Copyrights and reproduction rights","asset_fixed","False"
+"l10n_ie_account_212258","212258","Other similar rights and assets created by the undertaking itself","asset_fixed","False"
+"l10n_ie_account_213","213","Goodwill acquired for consideration","asset_fixed","False"
+"l10n_ie_account_214","214","Down payments and intangible fixed assets under development","asset_fixed","False"
+"l10n_ie_account_221111","221111","Developed land","asset_fixed","False"
+"l10n_ie_account_221112","221112","Property rights and similar","asset_fixed","False"
+"l10n_ie_account_221118","221118","Other land","asset_fixed","False"
+"l10n_ie_account_22112","22112","Land in foreign countries","asset_fixed","False"
+"l10n_ie_account_22121","22121","Fixtures and fitting-outs of land in Ireland","asset_fixed","False"
+"l10n_ie_account_22122","22122","Fixtures and fitting-outs of land in foreign countries","asset_fixed","False"
+"l10n_ie_account_221311","221311","Residential buildings","asset_fixed","False"
+"l10n_ie_account_221312","221312","Non-residential buildings","asset_fixed","False"
+"l10n_ie_account_221313","221313","Mixed-use buildings","asset_fixed","False"
+"l10n_ie_account_221318","221318","Other buildings","asset_fixed","False"
+"l10n_ie_account_22132","22132","Buildings in foreign countries","asset_fixed","False"
+"l10n_ie_account_22141","22141","Fixtures and fitting-outs of buildings in Ireland","asset_fixed","False"
+"l10n_ie_account_22142","22142","Fixtures and fitting-outs of buildings in foreign countries","asset_fixed","False"
+"l10n_ie_account_22151","22151","Investment properties in Ireland","asset_fixed","False"
+"l10n_ie_account_22152","22152","Investment properties in foreign countries","asset_fixed","False"
+"l10n_ie_account_2221","2221","Plant","asset_fixed","False"
+"l10n_ie_account_2222","2222","Machinery","asset_fixed","False"
+"l10n_ie_account_2231","2231","Transportation and handling equipment","asset_fixed","False"
+"l10n_ie_account_2232","2232","Motor vehicles","asset_fixed","False"
+"l10n_ie_account_2233","2233","Tools","asset_fixed","False"
+"l10n_ie_account_2234","2234","Furniture","asset_fixed","False"
+"l10n_ie_account_2235","2235","Computer equipment","asset_fixed","False"
+"l10n_ie_account_2236","2236","Livestock","asset_fixed","False"
+"l10n_ie_account_2237","2237","Returnable packaging","asset_fixed","False"
+"l10n_ie_account_2238","2238","Other fixtures","asset_fixed","False"
+"l10n_ie_account_22411","22411","Land, fitting-outs and buildings in Ireland","asset_fixed","False"
+"l10n_ie_account_22412","22412","Land, fitting-outs and buildings in foreign countries","asset_fixed","False"
+"l10n_ie_account_2242","2242","Plant and machinery","asset_fixed","False"
+"l10n_ie_account_2243","2243","Other fixtures and fittings, tools and equipment (including rolling stock)","asset_fixed","False"
```

Comment:
These accounts seem redundant since their asset categories are already covered by accounts 221-223.
So I would suggest removing them. Or alternatively, keep them, but remove accounts 221 to 223.